lib/resources/example_json/soarian_examples_immunization.rb

```diff
@@ -0,0 +1,249 @@
+module Cerner
+  module Resources
+
+    SOARIAN_IMMUNIZATION_BUNDLE ||=
+     {
+    "resourceType": "Bundle",
+    "id": "debcb450-dcf0-492d-8768-eb8360c159e2",
+    "meta": {
+        "lastUpdated": "2017-02-28T11:15:03.930-05:00"
+    },
+    "extension": [
+        {
+            "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+            "valueExtension": {
+                "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "papi009"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "This is the most current, known Immunization list and may not reflect the state of immunizations known at a previous date."
+                    }
+                ]
+            }
+        }
+    ],
+    "type": "searchset",
+    "link": [
+        {
+            "relation": "self",
+            "url": "base-url/123abc/Immunization?patientId=52930295-B52D-4685-B922-DC5F181CF786&_format=json"
+        }
+    ],
+    "entry": [
+        {
+            "resource": {
+                "resourceType": "Immunization",
+                "text": {
+                    "status": "additional",
+                    "div": "&lt;li ccdRefId=\'ZImmunizations.Immunizations.10400115510\'>&lt;b>FLU VAC TV 2015(18 YR+)RCM/PF&lt;/b> (FLUBLOK 2015-2016, ABBVIE US LLC, Lot # 124578AQ); Administered 2/28/2017 11:16 AM; 135 MCG = 0.5 ML, Intramuscular&lt;/li>"
+                },
+                "status": "Completed",
```

Comment:
The code returns "Completed" whereas it should be "completed" per FHIR. This will have to go in as a bug fix.  @daliboz, this most likely can't be done before certification.  To ensure that our documentation matches the code, I propose that we update both post-certification.  Are you ok with that approach?